### DISCUSSION TITLE
Interface Bulk Import

### DISCUSSION THREAD

**Discussion by ISAYLG** [netbox-community/netbox#7934] [Q&A]
I'm trying to update one of my device template and need to update the interface type to qsfp(40ge). using the csv data, I put device,name,type and then type as qsfp+ (40ge) but I got error to select a valid choice and that qsfp+ (40ge)  is not valid!

Also if I want, at the same time, to assign the interface to a port-channel and then add the vlans, what do I need to add to the csv data?
(for example device,name,type,lag,...)

**Comment by Kludge-As-A-Service** [comment]
Regarding your first problem, go to the interfaces menu on the left pane => import => click on the "?" icon on the left in the "type" row, Then you'll see all legal types in the left column. In your case I suppose it would be 40gbase-x-qsfpp.
